ci/run_tests.sh

```diff
@@ -1,4 +1,15 @@
 #!/bin/sh
 set -evx
 
-chef-server-ctl test -J $WORKSPACE/pedant.xml --all
+sudo chef-server-ctl test -J $WORKSPACE/pedant.xml --all
```

Comment:
While it may be true that the `test` command doesn't need root (although, it may need to run as the service user to read some config), the chef-server-ctl command itself has a simple check:

https://github.com/chef/chef-server/blob/master/omnibus/config/software/private-chef-ctl.rb#L65-L69

so in practice you need to run all of its commands as root.